README.md

```diff
@@ -1,2 +1,174 @@
+ - [![bage][http atf policy bage]][http atf policy]
+
+#### Automated RC regression 
+
+#### Various features regression
+Contains all ATF scripts for all featured available in the development.
+For parallel execution, checks are split to multiple jobs with make name template **Develop_TCP_ATF_VF{X}_{P,H,E}**. **X** is the number.
+
+Full list of regression ATF jobs available on : https://opensdl-jenkins.prjdmz.luxoft.com/view/ATF_Regression_on_TCP/ 
+
+## PR to develop checks
+
+Pull request jobs are available in view: https://opensdl-jenkins.prjdmz.luxoft.com/view/PR_checks/ 
+
+PR checks : 
+ - code style
+ - build with unit tests
+ - [smoke_tests.txt](https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/master/test_sets/smoke_tests.txt) trigger by build with unit tests.
+
+## Weekly checks 
+
+List of develop jobs executed [weekly](https://opensdl-jenkins.prjdmz.luxoft.com/view/Develop_weekly/)
+
+### Unit tests coverage 
+
+Unit tests coverage check weekly in the job: [![bage][unit test coverage bage]][unit test coverage]
+
+### Full ATF regression
+
+On Saturday, SDL CI performs a full regression check on these following transports: TCP, WebSockets, SecureWebSockets, and all 3 policy flow. 
+Weekly builds are available on the view https://opensdl-jenkins.prjdmz.luxoft.com/view/Develop_weekly/ 
+
+Each SDL build without unit tests triggers full ATF regression. 
+List of triggered jobs available in each build job as Downstream projects, example:
+https://opensdl-jenkins.prjdmz.luxoft.com/view/all/job/Develop_SDL_NoUT_E_BWSS_OFF/ 
+
+Full weekly status is available on https://opensdl-jenkins.prjdmz.luxoft.com/view/Develop_weekly_status/ 
+
+## Feature checks:
+
+For each feature before merging to develop should be created a list of jobs similar to develop to check that feature will no introduce a regression. 
+There is a special job in CI [Feature job create]() that will create a list of jobs and a separate view for the feature.
+
+Required input values for feeature job: 
+ - Feture name (will be used for view title)
+ - sdl_core feature branch and repository (master by default) 
+ - sdl_atf feature branch and repository (master by default)
+ - sdl_atf_test_scripts deature branch and repository. (master by default)
+ - Feature test set (optional)
+ - Additional info : evoluiton proposal, links to issues, etc ...
+ 
+After job execution will be created a view with the following checks:
+ 1. SDL build with unit tests on 3 policy flows (triggers: push, nightly)
+ 2. SDL build without unit tests on 3 policy flows (triggers: push, nightly)
+ 3. Smoke tests on 3 policy flows (triggers: build jobs without unit tests)
+ 4. Feature tests on 3 policy flows (triggers: build jobs without unit tests)
+ 5. Regression tests on 3 policy flows (triggers: build jobs without unit tests)
+
+## Special requests
+
+In case of special requiest (not ordinaary builds for special feature, restart certain jobs, etc, ...) please contact any of following persons:
```

Comment:
Please fix typos `requiest` and `ordinaary`

Reply:
@YarikMamykin done in https://github.com/LuxoftSDL/sdl_core_ci/pull/1/commits/a280e8231a579ed768b23a5aa05ba0f1c1e3b2e6